### PR TITLE
Update title of Customized Accordion section of Accordion demo (#5672)

### LINKED ISSUE #5672: Update the title of the `Customized Accordion` section of the `Accordion` demo page
The current title of this section is too long and not appropriate:

image

This title should be changed to `Customization`.

The demo file name is `BitAccordionDemo.razor` in the `Bit.BlazorUI.Demo.Client.Core` project.

### PR DESCRIPTION
**Pull request by MahdiAbbasi7**
this closes #5672

### CODE REVIEW

File: src/BlazorUI/Demo/Client/Core/Pages/Components/Accordion/BitAccordionDemo.razor [hunk 1]
```diff
@@ -139,4 +139,4 @@
     private byte controlledAccordionExpandedItem = 1;
     private bool AccordionToggleIsEnabled;
     private bool AccordionToggleIsExpanded;
-}
```
Comment (msynk) [inline src/BlazorUI/Demo/Client/Core/Pages/Components/Accordion/BitAccordionDemo.razor:142]: please revert this change, do not remove this empty new line at the end of the file.

**Review by msynk — CHANGES_REQUESTED** [review]

**Comment by MahdiAbbasi7** [comment]
oops... Is that Ok now?

**Review by msynk — APPROVED** [review]
